tests/test_battery_performance.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+from src.check_limits import BatteryManagementSystem
+
+
+class TestBatteryPerformance(TestCase):
+
+    def setUp(self) -> None:
+        self.battery_obj = BatteryManagementSystem()
+
+    def test_battery_is_ok(self):
```

Comment:
Test cases need to be strengthened further. At least minimum set of permutation and combinations of all three parameter values should be considered. For example one parameter will have 2 boundary values and 3 in between values, i.e. <, ==, > threshold values.